### ISSUE TITLE
gateway error what did i mess up? or is it broken?

### ISSUE THREAD

**Issue by 3BixxPy** [hikari-py/hikari#1068]
### Steps to reproduce

```
import hikari
bot = hikari.GatewayBot(token="my token")
bot.run() 
```

### Expected result

bot should be online

### Actual result

bot still offline

### System info

```shell
I 2022-03-09 17:28:30,916 hikari.bot: you can start 1000 sessions before the next window which starts at 2022-03-09 17:28:30.916942+01:00; planning to start 1 session... 
I 2022-03-09 17:28:35,492 hikari.gateway.0: shard has disconnected and shut down normally
I 2022-03-09 17:28:35,494 hikari.bot: successfully terminated
Traceback (most recent call last):
  File "C:\Users\user\OneDrive\pc\folders\discord_bot_py\bottest.py", line 4, in <module>
    bot.run()
  File "C:\Users\user\OneDrive\pc\folders\discord_bot_py\venv\lib\site-packages\hikari\impl\bot.py", line 878, in run
    loop.run_until_complete(
  File "C:\Users\user\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 641, in run_until_complete
    return future.result()
  File "C:\Users\user\OneDrive\pc\folders\discord_bot_py\venv\lib\site-packages\hikari\impl\bot.py", line 1072, in start
    await aio.all_of(
  File "C:\Users\user\OneDrive\pc\folders\discord_bot_py\venv\lib\site-packages\hikari\internal\aio.py", line 193, in all_of
    await gatherer  # pragma: no cover
  File "C:\Users\user\OneDrive\pc\folders\discord_bot_py\venv\lib\site-packages\hikari\internal\aio.py", line 174, in all_of
    return await asyncio.wait_for(gatherer, timeout=timeout)
  File "C:\Users\user\AppData\Local\Programs\Python\Python310\lib\asyncio\tasks.py", line 408, in wait_for
    return await fut
  File "C:\Users\user\OneDrive\pc\folders\discord_bot_py\venv\lib\site-packages\hikari\impl\bot.py", line 1388, in _start_one_shard
    raise errors.GatewayError(f"shard {shard_id} shut down immediately when starting")
hikari.errors.GatewayError: shard 0 shut down immediately when starting
```

### Further info

my bot is setup correctly. it worked yesterday just fine

should have put a help label instead

### Checklist

- [X] I have made sure to remove ANY sensitive information (bot token, passwords, credentials, personal details, etc.).
- [X] I have searched the issue tracker and have made sure it's not a duplicate. If it is a follow up of another issue, I have specified it.

**Comment by HyperGH** [comment]
I'm not entirely certain what is causing this issue specifically, but I would recommend you join the [hikari discord](https://discord.gg/Jx4cNGG) for faster and easier troubleshooting!

**Comment by 3BixxPy** [comment]
still no fix after 2 hours. i gave up

**Comment by FasterSpeeding** [comment]
This issue is only on the master branch and to avoid it you can just use a pypi release. There's already a fix for it in #1061
